src/dbos.ts

```diff
@@ -226,13 +226,27 @@ export class DBOS {
     // Initialize the DBOS executor
     if (!DBOS.dbosConfig) {
       const [dbosConfig, runtimeConfig] = parseConfigFile({ forceConsole: isDebugging });
-      if (!isDebugging) {
+      if (!isDebugging && dbosConfig.poolConfig) {
+        dbosConfig.poolConfig = await db_wizard(dbosConfig.poolConfig);
+      }
+      DBOS.dbosConfig = dbosConfig;
+      DBOS.runtimeConfig = runtimeConfig;
+    } else if (!isDeprecatedDBOSConfig(DBOS.dbosConfig)) {
+      let [dbosConfig, runtimeConfig] = translatePublicDBOSconfig(DBOS.dbosConfig);
```

Comment:
Seems like translatePublicDBOSconfig needs a forceConsole parameter to correctly configure `dbosConfig.telemetry.logs.forceConsole`